recipes/simple-websocket-server/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "v2.0.0":
```

```suggestion
  "2.0.0":
```

Please, discard the prefix

---

Even if the main project used those?